new study: ccrcc_wcm_2022

alexsigaras opened the pull request:
# What?
Added new clear cell renal carcinoma (ccRCC) study. The paper is accepted and the PMID and citation are available.

# checks
For all pull requests:
- [x] Passes validation

For a new study (in addition to above):
- [x] Does study name and study ID follow our convention? e.g. Tumor_Type (Institue, Journal Year); brca_mskcc_2015
- [x] is study meta data complete? e.g. pmid, group of PUBLIC
- [x] were all samples profiled with WES/WGS? If not, is gene panel file curated?
- [ ] are oncotree codes of all samples curated; Cancer Type and Cancer Type Detailed needs to be added in addition to Oncotree Code
- [x] clinical sample and patient data with meta files
- [x] mutations data with meta files
- [x] MAF is based on hg19
- [ ] MAF with 2 isoforms: uniprot and mskcc
- [x] CNA data with meta files
- [ ] CNA segment data with meta files
- [ ] Expression data including z-scores with meta files
- [x] Case-lists for all profiles.
- [ ] Manual checking (Niki or JJ): Triage or private Portal link here

alexsigaras committed:
new study: ccrcc_wcm_2022

alexsigaras committed:
updated groups to PUBLIC

rmadupuri commented:
Hi @alexsigaras, thank you for the PR. The data formats look good! Some fixes are required before we can merge,
1. Can you add the Oncotree Codes to the clinical sample file?
2. Does all the samples have a matched normal? Can this info be added to clinical file as well? The attribute we use is SOMATIC_STATUS (Matched/Unmatched).
3. The paper mentions 68 samples from 44 patients. And the data here is for 67 samples from 43 patients. Can you add to the description on why one patient was excluded in the portal?
4. Is the MAF annotated with Genome Nexus?
5. Can you rename the `meta_mutation_extended.txt` file to `meta_mutations.txt`
6. A few case lists are missing. Can you generate them using the script here - https://github.com/cBioPortal/datahub-study-curation-tools/tree/master/generate-case-lists

rmadupuri commented:
Testing instance of the study: https://triage.cbioportal.mskcc.org/study/summary?id=ccrcc_wcm_2022

alexsigaras committed:
Updated cohort based on feedback in PR

alexsigaras committed:
Merge branch 'master' into ccrcc_wcm_2022

alexsigaras commented:
@rmadupuri: We have updated the cohort based on the feedback requested and have also updated the branch to match master. Could you please take another look and let us know if everything looks ok to merge?

Thanks,
Alex

rmadupuri commented:
Thank you for the updates @alexsigaras. The fixes look good! 

One additional issue that we noticed with the MAF - there are ~500 variants with either the position is off by 1 or the alleles are incorrect. The annotation failed on these variants and they would show up incorrectly in the portal. I am attaching the problematic variants here - [data_mutations_unannotated.txt](https://github.com/cBioPortal/datahub/files/8475820/data_mutations_unannotated.txt). 

Could you double check these and give us the right coordinates/alleles? Let us know if any issues.

Thanks!

rmadupuri commented:
HI @alexsigaras just checking, do you have an update for the above issue? Thanks!

rmadupuri committed:
Merge branch 'master' into ccrcc_wcm_2022

alexsigaras committed:
Merge branch 'master' into ccrcc_wcm_2022

Commit:
Merge branch 'ccrcc_wcm_2022' of github.com:cBioPortal/datahub

Commit:
Adding new public study

alexsigaras committed:
Merge branch 'master' into ccrcc_wcm_2022

alexsigaras committed:
Merge branch 'master' into ccrcc_wcm_2022

rmadupuri commented:
Duplicated in #1892
